pkg/constants/constants.go

```diff
@@ -275,3 +275,6 @@ const VMwareSystemVMUUID = "vmware-system-vm-uuid"
 
 // Label to set for Velero to ignore resources
 const VeleroExcludeLabel = "velero.io/exclude-from-backup"
+
+// Maximum time window to clean up CR which is in terminate state
+const CRCleanUpMaximum = 24
```

Comment:
This should be default time window, not maximum.

Reply:
Rename CRCleanUpMaximum to DefaultCRCleanUpWindow.